OWNERS_ALIASES

```diff
@@ -212,13 +212,17 @@ aliases:
     - seans3
     - soltysh
   sig-testing-reviewers:
+    - aojea
     - bentheelder
     - cblecker
-    - spiffxp
     - dims
+    - pohly
+    - spiffxp
   sig-testing-approvers:
```

Comment:
I think we should deprecate this alias instead, it only has a few people and we don't need to apply it to many directories.

Reply:
(And aliases obscure ownership)

Reply:
I agree and removed it in the first commit of this PR, then added me in the second.

Reply:
@dims: do you agree? Can you re-add the LGTM and perhaps approval?